### PR TITLE
Feat/met 2281 rewrite token info scheduled job

### PR DESCRIPTION
**Pull request by Sotatek-DucPhung**

### CODE REVIEW

File: src/test/java/org/cardanofoundation/job/service/impl/MultiAssetServiceImplTest.java [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
+  //
+  // Mockito.when(latestTokenBalanceRepository.countHoldersByMultiAssetIdInRange(multiAssetUnitIds))
+  //        .thenReturn(
+  //            Arrays.asList(
+  //                new TokenNumberHolders("unit6", 20L), new TokenNumberHolders("unit7", 25L)));
+  //
+  //    Map<String, Long> result = multiAssetService.getMapNumberHolderByUnits(multiAssetUnitIds);
+  //    assertEquals(20, result.get("unit6").longValue());
+  //    assertEquals(25, result.get("unit7").longValue());
+  //  }
```
Comment (Sotatek-DucPhung) [inline src/test/java/org/cardanofoundation/job/service/impl/MultiAssetServiceImplTest.java:26]: @Sotatek-ThinhVu this test unused anymore ?

File: .env.example [hunk 2]
```diff
@@ -41,6 +41,8 @@ UNIQUE_ACCOUNT_FIXED_DELAY=300000
 STAKE_TX_BALANCE_FIXED_DELAY=300000
 AGGREGATE_POOL_INFO_FIXED_DELAY=300000
 TOKEN_INFO_FIXED_DELAY=60000
+TOKEN_INFO_NUM_SLOT_INTERVAL=2000
```
Comment (Sotatek-DucPhung) [inline .env.example:44]: add this env in README as well

File: src/test/java/org/cardanofoundation/job/service/impl/MultiAssetServiceImplTest.java (hunk 1 above)
Comment (thinhvd200617) [reply src/test/java/org/cardanofoundation/job/service/impl/MultiAssetServiceImplTest.java]: updated!!!

File: .env.example (hunk 2 above)
Comment (Sotatek-ThinhVu) [reply .env.example:44]: updated!!!

File: src/test/java/org/cardanofoundation/job/service/impl/MultiAssetServiceImplTest.java (hunk 1 above)
Comment (Sotatek-ThinhVu) [reply src/test/java/org/cardanofoundation/job/service/impl/MultiAssetServiceImplTest.java]: updated!!!

**Pull request merged**